apps/explorer/src/ui/stories/Button.stories.tsx

```diff
@@ -1,10 +1,12 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+import { CheckFill16 } from '@mysten/icons';
 import { type StoryObj, type Meta } from '@storybook/react';
 import { MemoryRouter } from 'react-router-dom';
 
 import { Button, type ButtonProps } from '../Button';
+import { ReactComponent as CallIcon } from '../icons/transactions/call.svg';
```

Comment:
Can we just use the icons from the icons lib instead?

Reply:
This is only for the storybook, does I import both to show it works either way. But I can update.

Reply:
Yeah, we still want it to change because we're moving away from imported SVGs like this

Reply:
updated